SWIPs/swip-honey_to_money.md

```diff
@@ -7,51 +7,66 @@ Type: Standards track
+- The requirement on connectivity in the local bin and far-away bins means that nodes *must agree* with all nodes in their local bin and at least one node in a far-away bin on the price.
+- Zero marginal costs of serving data with perfect competition means that when nodes would engage in peer-to-peer price discovery, the price will tend to zero and a sub-optimal equilibrium of near-zero prices will be reached.
+- The high-quantity of transactions implies that nodes cannot engage in price-discovery for every byte of data transacted at the point of transaction, meaning they have to agree *beforehand* on how they will come to a price *at a later time*. 
+- Nodes are most of the time just conduits for data and payments, meaning that most the time data requests and thus also payments will be because you want to send data to a peer who requested this data from you. 
+### Rationale for the described oracle specification
+* Nodes might apply different honey prices for the same payment, due to the time difference between querying the oracle for *sending* a payment and querying the oracle for *receiving* the same payment. The imbalances caused by this don't harm if the sum of all inbalances is not greater than the disconnect threshold. Experimentation should confirm that this indeed does not happen. Theoretically, if the overall price movement does not show a clear trend (either up or down) over the period where nodes do business with each other or the two nodes are neither a net receiver nor a net sender to each other, accounting imbalances should offset each other. Furthermore, less frequent as well as small price updates should minimize the imbalances. If a clear trend exists and one node is a net user and the other a net provider, the disconnect threshold will eventually be reached. Research must be performed on how the strength of a trend influences the duration until a disconnect is reached for a given stream of traffic. The assumption is that this will be sufficiently long not to cause any problems for a typical Swarm user. If there appears to be a problem with such imbalances, we see a possible solution:
+As nobody benefits from price imbalances, both the sender and the receiver of a payment are incentivized to and can apply a price which is not per their clock, but per the clock of their counterparty when they expect their counterparty to query or have queried the oracle. To assist in this, a payload might be sent with the transaction, specifying the clock with which the oracle price was applied.
+* An on-chain oracle *might* give a different answer to two nodes when queried at the same time if there exists a (temporary) fork in the blockchain. Again: the logic applies here about the severity of such accounting imbalances. If this appears to be severe, any oracle price-update might be only considered valid if it happened a sufficient amount of time in the past (usually 12 blocks in Ethereum)
```

Comment:
i don't understand what `the logic applies here about the severity of such accounting imbalances` means. 

can you please explain?

Reply:
Basically what I wrote under:
`Rationale for the described oracle specification`.
If this is not clear, I will update it!

Reply:
let's get another opinion then, to see if this is indeed unclear or not :)

pinging @holisticode @vojtechsimetka @diegomasini

Reply:
My understanding is that for the case of imbalances generated by temporary forks in the blockchain the previous analysis and logic used for differences in honey price for a given payment applies. Probably it will be best to rephrase the sentence: "In this case the same logic described in this section about the severity of such accounting imbalances applies here."